packages/spotlight/src/spotlight.js

```diff
@@ -504,6 +504,13 @@ const Spotlight = (function () {
 		}
 	}
 
+	function onTouchEnd (evt) {
+		const current = getCurrent();
+		if (current && !current.contains(evt.target)) {
+			current.blur();
```

Comment:
What are the chances this will affect code that manually handles spotlight when dismissing?  If we blur here, will we prevent code like that in `ExpandableInput` or `ExpandableItem` that tries to figure out how to set the focus?  I'll try to do some more testing in that area but if you have some thoughts it'd be good to hear.

Reply:
Most of that code already deals with blur events as triggers so we should be okay. This was tested against Input and ExpandableInput for that reason.